_includes/manuals/3.3/1.2_release_notes.md

```diff
@@ -38,7 +36,238 @@ We've updated our browsers compatibility to support the following:
 * Mozilla Firefox - latest version
 * Mozilla Firefox Extended Support Release (ESR) - latest version
 
-### Release Notes
+### Release notes for 3.3.0
+#### Foreman
+* Support EC (and other non-RSA) keys in foreman core ([#34843](https://projects.theforeman.org/issues/34843))
+* Remove Share your feedback link in new host page
+ ([#34841](https://projects.theforeman.org/issues/34841))
+* Add host page reports tab  ([#34782](https://projects.theforeman.org/issues/34782))
+* Add developer documentation for pagelets ([#34544](https://projects.theforeman.org/issues/34544))
+* Host list page should show the host comment ([#34516](https://projects.theforeman.org/issues/34516))
+* Host list page should show the owner of the host ([#34515](https://projects.theforeman.org/issues/34515))
```

Comment:
Same here. 
Do or do not, there is no should ;-)

Reply:
> Same here. Do or do not, there is no should ;-)

Include the host's owner to a host list page
?

Reply:
That will work :)